domains/noctra.is-a-fullstack.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Noctra's Fullstack Domain",
+    "domain": "is-a-fullstack.dev",
+    "subdomain": "noctra",
+
+    "owner": {
+        "repo": "https://github.com/not-amplify/free-domain/",
+        "email": "not.amplify@night-x.com"
+    },
+
+    "record": {
+        "NS" : ["summer.ns.cloudflare.com", "terry.ns.cloudflare.com" ]
```

Comment:
Provide additional reasoning as to why you need NS as opposed to normal records.

Reply:
> Provide additional reasoning as to why you need NS as opposed to normal records.

so I can share it over cloudflare, and give him access to what he wants to add.

Reply:
> Provide additional reasoning as to why you need NS as opposed to normal records.

NS records are no longer allowed

Reply:
> > Provide additional reasoning as to why you need NS as opposed to normal records.
> 
> NS records are no longer allowed

why is that?

Reply:
> > > Provide additional reasoning as to why you need NS as opposed to normal records.
> > 
> > 
> > NS records are no longer allowed
> 
> why is that?

Unfortunately someone abused an NS record